toolkit/tools/imagecustomizerapi/kernelcommandline.go

```diff
@@ -5,13 +5,15 @@ package imagecustomizerapi
 
 type KernelCommandLine struct {
 	// Extra kernel command line args.
-	ExtraCommandLine KernelExtraArguments `yaml:"extraCommandLine"`
+	ExtraCommandLine []KernelExtraArguments `yaml:"extraCommandLine"`
```

Comment:
You can change `KernelExtraArguments` to `string`.

The purpose of this change is to have MIC handle the arg quoting. And the purpose of `KernelExtraArguments` is to check if the user has correctly quoted the args. Therefore, `KernelExtraArguments` isn’t needed anymore.